### PR TITLE
added if statement to check if total tags is more than 100, then for …

### PR DESCRIPTION
**Pull request by dalyd14**
…loop will make calls to populate rest of tags

## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/active_campaign/transform.js [hunk 1]
```diff
@@ -131,6 +131,48 @@ const customTagProcessor = async (message, category, destination) => {
     res.data.tags.map(t => {
       storedTags[t.tag] = t.id;
     });
+    const promises = [];
```
Comment (utsabc) [inline v0/destinations/active_campaign/transform.js:134]: Please add inline documentation about this

File: v0/destinations/active_campaign/transform.js [hunk 2]
```diff
@@ -131,6 +131,48 @@ const customTagProcessor = async (message, category, destination) => {
...
+        let i = 0;
+        i < Math.floor(parseInt(res.data.meta.total, 10) / 100);
+        i++
+      ) {
+        try {
+          const resp = axios.get(
+            `${destination.Config.apiUrl}${
+              category.tagEndPoint
+                ? `${category.tagEndPoint}?limit=100&offset=${100 * (i + 1)}`
+                : ""
```
Comment (shrouti1507) [inline v0/destinations/active_campaign/transform.js:150]: if without `category.tagEndPoint` it is not possible to use `limit` or `offset` in query params, can we check it's availability previously? I think then the line of code will be a bit easy to read.

Comment (utsabc) [reply v0/destinations/active_campaign/transform.js:150]: We can actually remove the ternary condition as `tagEndPoint` will always be present in category

Comment (utsabc) [reply v0/destinations/active_campaign/transform.js:150]: @yashasvibajpai Check once

File: v0/destinations/active_campaign/transform.js (hunk 1 above)
Comment (yashasvibajpai) [reply v0/destinations/active_campaign/transform.js:134]: Added

**Review by utsabc — APPROVED** [review]

**Review by akashrpo — APPROVED** [review]

**Pull request merged**